server/controllers/common/signoutHandler.js

```diff
@@ -0,0 +1,14 @@
+const { throwError } = require('../../utilities');
+
+const signoutHandler = (req, res, next) => {
+  try {
+    res.clearCookie('token');
+    res.json({
+      title: 'Logout Successful',
+    });
+  } catch (err) {
+    next(throwError(400, 'logOut field'));
+  }
+};
```

Comment:
the try-catch doesn't work here, 
you can check cookies from the request, if they exist you can clear them, if not you send a message like 'you'r not log in'